components/channel_header_dropdown/channel_header_dropdown_items.tsx

```diff
@@ -111,13 +115,11 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                 />
                 <Menu.Group divider={divider}>
                     <MenuItemToggleFavoriteChannel
-                        id='channelToggleFavorite'
```

Comment:
We most probably will be using ID for e2e, please put them back

Reply:
Props of `MenuItemToggleFavoriteChannel` doesn't define `id`.
I fixed this in 98696dff5e7385c05d7d139c4841436e2a79b82a, however, this is out of scope of this PR.
I reset  98696dff5e7385c05d7d139c4841436e2a79b82a's changes.
I needed to remove `id` so that `make check-types` does not emit any error.

Reply:
yes the ids were not being passed down the component, so i guess there were not used in the e2e. but still we would run e2e test to ensure nothings breaks